packages/datepickers/src/styled/StyledHeaderPaddle.ts

```diff
@@ -45,25 +71,19 @@ const COMPONENT_ID = 'datepickers.header_paddle';
 export const StyledHeaderPaddle = styled.div.attrs({
   'data-garden-id': COMPONENT_ID
 })<{
-  isCompact: boolean;
-  isHidden?: boolean;
+  $isCompact: boolean;
+  $isHidden?: boolean;
 }>`
   display: flex;
   align-items: center;
   justify-content: center;
   transform: ${props => props.theme.rtl && 'rotate(180deg)'};
-  visibility: ${props => props.isHidden && 'hidden'};
+  visibility: ${props => props.$isHidden && 'hidden'};
```

Comment:
This smells like it should be `[aria-hidden]` CSS rather than passing in a `$isHidden` prop.